vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -325,6 +325,7 @@
     <Import Project="$(SolutionDir)\packages\Microsoft.Build.Tasks.Git.1.0.0\build\Microsoft.Build.Tasks.Git.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.Build.Tasks.Git.1.0.0\build\Microsoft.Build.Tasks.Git.targets')" />
     <Import Project="$(SolutionDir)\packages\Microsoft.SourceLink.Common.1.0.0\build\Microsoft.SourceLink.Common.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.SourceLink.Common.1.0.0\build\Microsoft.SourceLink.Common.targets')" />
     <Import Project="$(SolutionDir)\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.targets')" />
+    <Import Project="$(SolutionDir)\packages\XamlStraddle.0.0.3\build\XamlStraddle.targets" Condition="Exists('$(SolutionDir)\packages\XamlStraddle.0.0.3\build\XamlStraddle.targets')" />
```

Comment:
>Import [](start = 5, length = 6)

The desktop project must not depend on XAML.

Reply:
Agree but it does today. Feel free to file/fix though :-)